src/plugins/kernel_install/raspbian.sh

```diff
@@ -0,0 +1,33 @@
+# Kworkflow treats this script as a plugin for installing a new Kernel or
+# module in a target system. It is essential to highlight that this file
+# follows an API that can be seen in the "deploy.sh" file, if you make any
+# changes here, you must adjust install_modules() and install_kernel()
+# accordingly.
+#
+
+# Update boot loader API
+function update_raspbian_boot_loader()
+{
+  local name="$1"
+  local target="$2"
+  local flag="$3"
+
+  # https://www.raspberrypi.org/documentation/configuration/config-txt/boot.md
+
+  if grep --quiet "kernel" /boot/config.txt; then
+    sudo sed -ri "/kernel/s/=.+/=$name/g" /boot/config.txt
```

Comment:
I wonder what happens without a kernel option at config.txt. I guess we could assume it to be always present, though people may use diverse operating systems with raspberry or no OS at all.

Reply:
@marceloschmitt1 From what I could ascertain, if there is not a kernel parameter in the config.txt the default value is used (which is board dependent)